gematria/testing/python/ir_utils.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for writing tests that work with IR.
+
+This library contains several functions that perform functions such as converting
+between LLVM Bitcode and textural IR to enable more easily writing tests that
+work with these formats.
+"""
+
+import subprocess
+import os
+from collections.abc import Sequence
+
+
+import pandas
+import pyarrow
+from pyarrow import parquet
+from rules_python.python.runfiles import runfiles
+
+
+def _get_llvm_binary_path(tool_name: str) -> str:
+  runfiles_env = runfiles.Create(os.environ)
+  assert runfiles_env is not None
+  return runfiles_env.Rlocation('llvm-project/llvm/' + tool_name)
+
+
+def get_bc_from_ir(ir: str) -> bytes:
+  """Generates bitcode for an IR string.
```

Comment:
Very nit: Add an empty line between the first (summary) line of the docstring and the rest.

Same in the other docstrings.